io/disk/tiobench.py

```diff
@@ -44,9 +44,13 @@ def setUp(self):
         Source:
         https://github.com/mkuoppal/tiobench.git
         """
-        s_mngr = SoftwareManager()
-        if not s_mngr.check_installed("gcc") and not s_mngr.install("gcc"):
-            self.error('Gcc is needed for the test to be run')
+        smm = SoftwareManager()
+        if distro.detect().name == 'Ubuntu':
+            if not smm.check_installed("btrfs-tools") and not \
+                    smm.install("btrfs-tools"):
+                self.skip('btrfs-tools is needed for the test to be run')
+        if not smm.check_installed("gcc") and not smm.install("gcc"):
+            self.skip('Gcc is needed for the test to be run')
```

Comment:
btrfs package should be installed only if `fstype` is btrfs.
Please refer: https://github.com/avocado-framework-tests/avocado-misc-tests/blob/master/io/disk/lvsetup.py